app-games/src/main/java/com/aptoide/android/aptoidegames/appview/AppViewScreen.kt

```diff
@@ -514,16 +520,66 @@ fun ScreenshotsList(screenshots: List<String>) {
           .clearAndSetSemantics {
             contentDescription = stringResource
           }
-          .size(268.dp, 152.dp),
-        data = screenshot,
+          .height(152.dp)
+          .aspectRatio(screenshot.width.toFloat() / screenshot.height.toFloat()),
+        data = screenshot.url.toImageUrl(density, densityDpi, screenshot.height > screenshot.width),
         contentDescription = null,
         selectedIndex = index,
-        images = screenshots
+        images = screenshots.map { it.url },
+        contentScale = ContentScale.Fit,
       )
     }
   }
 }
 
+fun String.toImageUrl(density: Float, densityDpi: Int, isPortrait: Boolean): String {
```

Comment:
This could be moved to the extensions module together with the private functions and the name could mention that density is used

Reply:
Name fixed. Regarding moving it to the extensions module, for now it doesn't make sense since the function is only used here, if we extend it to use it elsewhere we should move it there